.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
+*.iml
```

Comment:
You should put that in your global .gitignore instead (but same could be said for `.idea` I guess :D